src/content/guides/minimal-startup.md

```diff
@@ -0,0 +1,389 @@
+
+__project__
+``` diff
+  webpack-basics
+  |- indexMain.html
+  |- entryMain.js
++ |- indexBundle.html
++ |- src
++   |- entryBundle.js
+  |- dist
+    |- main.js
+    |- bundle.js
+```
+
+After copying the files we will need to edit indexBundle.html to point to the new bundled file, dist/bundle.js. It's text should also be updated to reflect it is indexBundle.html instead of indexMain.html. Similarly modify the console.log message in entryBundle.js.
+
+__webpack-basics/indexBasics.html__
+``` html
+<!DOCTYPE html>
+<html>
+  <body>
+    Webpack indexBundle.html Body Text (With only: script src='./dist/bundle.js').
+    <br> <script src='./dist/bundle.js'></script>
+  </body>
+</html>
+```
+
+__webpack-basics/src/entryBundle.js__
+``` JavaScript
+  console.log('Console.log message inside entryBundle.js')
+```
+
+
+Now rerun webpack.
+
+``` bash
+> webpack src/entryBundle.js -o dist/bundle.js --mode development
+
+Hash: ae8ebe3170f4c6373946
+Version: webpack 4.17.1
+Time: 1660ms
+Built at: 2018-08-25 11:31:33
+  Asset       Size  Chunks             Chunk Names
+bundle.js  984 bytes       0  [emitted]  main
+Entrypoint main = bundle.js
+[0] ./entryMain.js 56 bytes {0} [built]
+```   
+
+Change your browser URL to ...webpack-basics/indexBundle.html and you should now be able to see the updated text inside indexBundle.html as well as the updated console message from entryBundle.js.
+
+## Part III. Webpacking Multiple Files
+Webpack automatically "sucks in" and bundles multiple JavaScript files as needed.  The simplest way to do this is to use a require statement, such as require('two.js'). Webpack begins by bundling the entry JavaScript file found in the webpack command.  It then processes and bundles the required files found in this file and repeats.  In this manner all required file(s) are automatically bundled and correctly ordered along with the entry file.
+
+Copy entryBundle.js to entry.js and indexBundle.html to index.hmtl. Edit index.html to include div tags with text. The text inside the div tags will be updated in the JavaScript files using document.getElementById("xx").innerText = "text from .js file".
+
+__webpack-basics/index.html__
+``` html
+<!DOCTYPE html>
+<html>
+	<body>
+		<br> <div> Webpack index.html Body Text with 3 Divs </div>
+		<br> <div id="d1">Para1 set in index.html</div>
+		<br> <div id="d2">Para2 set in index.html</div>
+		<br> <div id="d3">Para3 set in index.html</div>
+		<br> <script src='./dist/bundle.js'></script>
+	</body>
+</html>
+```
+
+ Edit entry.js to include require statements and update the first div.
+
+__webpack-basics/src/entry.js__
+``` JavaScript
+require('./two.js')
+require('./three.js')
+console.log('Console message from entry.js')
+document.getElementById('d1').innerText='d1 set by entry.js'
+```
+
+Create two.js and thee.js in the src folder.  Edit them to include a console.log and an update of the divs.
+
+__webpack-basics/src/two.js__
+``` JavaScript
+console.log('Console message from two.js')
```

Comment:
examples are written with semicolons, and spaces for operators e.g. `var b = 1;`